src/chocolatey.tests/infrastructure.app/nuget/NugetCommonSpecs.cs

```diff
@@ -175,14 +178,14 @@ public void Should_set_user_agent_string()
                 Context();
                 var source = "https://community.chocolatey.org/api/v2/";
                 _configuration.Sources = source;
-                _configuration.Information.ChocolateyProductVersion = "vNext";
 
                 _because();
 
                 // Change this when the NuGet version is updated.

```

Comment:
I think so, I'd assume this needs to be changed whenever we update the NuGet libraries.

Reply:
But the line that was 183 was removed from what I can tell. Now instead of building the user agent and including the NuGet Client Version, we're doing a regex...